apps/pwabuilder/package.json

```diff
@@ -28,13 +30,17 @@
     "colorjs.io": "^0.4.0",
     "cors": "^2.8.5",
     "express": "^4.17.3",
+    "html-to-image": "^1.11.4",
```

Comment:
Lets double check that we remove these deps after the pure canvas implementation is done